gripper_com_pose:
  - 0.002621568387731212
  - -0.0003515309468810598
  - 0.004137037404081813
  - 0
  - 0
  - 0
gripper_mass: 6.509143994937928e-07
bias:
  - 1.584460183819159e-08
  - -5.707289241210769e-06
  - 1.498097625527359e-06
  - 3.64819715407862e-08
  - -2.265979235962194e-08
  - 2.964348029762117e-09
gripper_com_frame_id: lwr_7_link